styleguide/source/assets/images/logo-reversed.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```

Comment:
i was thinking of what we’re doing with the icons, those are getting run through `svgmin` by gulp but you’re right, we’re not doing anything with the logos. Ignore me, carry on.